docs/yul.rst

```diff
@@ -976,6 +976,64 @@ within one Yul subobject. If at least one ``memoryguard`` call is found in a sub
 the additional optimiser steps will be run on it.
 
 
+verbatim
+^^^^^^^^
+
+The set of ``verbatim...`` builtin functions lets you create bytecode for opcodes
+that are not known to the Yul compiler. It also allows you to create
+bytecode sequences that will not be modified by the optimizer.
+
+The functions are ``verbatim_<n>i_<m>o("<data>", ...)``, where
+ - ``n`` is a decimal that specifies the number of input stack slots / variables
+ - ``m`` is a decimal that specifies the number of output stack slots / variables
+ - ``data`` is a string literal that contains the sequence of bytes
+
+If you for example want to define a function that multiplies the input
+by two, without the optimizer touching the constant two, you can use
+
+.. code-block:: yul
+
+    let x := calldataload(0)
+    let double := verbatim_1i_1o(hex"600202, x)
+
+This code will result in a ``dupN`` opcode to retrieve ``x``
```

Comment:
```suggestion
This code will result in a ``dup1`` opcode to retrieve ``x``
```
Maybe a bit clearer?

Reply:
It depends a little on x but yeah, we can write that.